app/code/Magento/Cms/Api/BlockManagementInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Cms\Api;
+
+/**
+ * @api
+ */
+
+interface BlockManagementInterface
+{
+    /**
+     * Load block data by given block identifier.
+     *
+     * @param string $identifier
+     * @param int|null $storeId
+     * @return \Magento\Cms\Api\Data\BlockInterface
+     */
+    public function getByIdentifier($identifier, $storeId = null);
```

Comment:
PHP 7.0 type hint syntax may be used from 2.2 version onwards. This declaration may be updated accordingly.